project-koku/koku
# COST-2152: Remove GCP Mat Views

TODO: RUN SMOKE TESTS
## How to test
1. Check out main and load db with gcp data
2. After the db has been loaded with gcp data check to ensure that the gcp mat views exist:
```
postgres=# \dm reporting_gcp*
                                  List of relations
  Schema   |                   Name                   |       Type        |  Owner
-----------+------------------------------------------+-------------------+----------
 acct10001 | reporting_gcp_compute_summary            | materialized view | postgres
 acct10001 | reporting_gcp_compute_summary_by_account | materialized view | postgres
 acct10001 | reporting_gcp_compute_summary_by_project | materialized view | postgres
 acct10001 | reporting_gcp_compute_summary_by_region  | materialized view | postgres
 acct10001 | reporting_gcp_compute_summary_by_service | materialized view | postgres
 acct10001 | reporting_gcp_cost_summary               | materialized view | postgres
 acct10001 | reporting_gcp_cost_summary_by_account    | materialized view | postgres
 acct10001 | reporting_gcp_cost_summary_by_project    | materialized view | postgres
 acct10001 | reporting_gcp_cost_summary_by_region     | materialized view | postgres
 acct10001 | reporting_gcp_cost_summary_by_service    | materialized view | postgres
 acct10001 | reporting_gcp_database_summary           | materialized view | postgres
 acct10001 | reporting_gcp_network_summary            | materialized view | postgres
 acct10001 | reporting_gcp_storage_summary            | materialized view | postgres
 acct10001 | reporting_gcp_storage_summary_by_account | materialized view | postgres
 acct10001 | reporting_gcp_storage_summary_by_project | materialized view | postgres
 acct10001 | reporting_gcp_storage_summary_by_region  | materialized view | postgres
 acct10001 | reporting_gcp_storage_summary_by_service | materialized view | postgres
(17 rows)
```
3. Switch back to this branch and run the migration file:
```
make run-migrations
```
4. Switch back to the db and check to see if the gcp mat views still exist:
```
postgres=# \dm reporting_gcp*
Did not find any relation named "reporting_gcp*".
```
